### ISSUE TITLE
Help regarding understanding of the licensing model

### ISSUE THREAD

**Issue by steffenstolze** [postalsys/emailengine#262]
Hey there,

looking for a solution to watch 100s of IMAP accounts and trigger actions when receiving mails in a commercial context, we thought about implementing an own solution based on node-fetch and stumbled across your amazing product which comes with all batteries included. 

Considering the information on https://postalsys.com/plans a commercial company could pay €695/year for a subscription based license to **use** the features that EmailEngine provides (reselling is **not** intended) for their business. Meaning, if you have a company that needs to process hundreds of IMAP inboxes and forward received mails to web hooks for example, this product could be used in a commercial context.

Could you please clarify these questions:

- Is the interpretation above correct?
- What does the "_12 team members_" limitation account for? 12 internal people operating the EmailEngine application - having application logins?
- Point 3.2 of the license agreement states: 
  - >Upon any termination of this Agreement, for any
reason, You will promptly cease use of the Commercial Features and destroy any
copies thereof
  - this reads like "We can change that license agreement at any time and you'd immediately not be allowed to use the software commercially anymore." and does not sound very promising if you want to integrate a commercial software product into your system architecture
  - did I get this right?

Thanks and BR

**Comment by andris9** [comment]
Hi Steffen,

Thank you for your interest in EmailEngine.

1. Yes, you are correct on the interpretation. EmailEngine has customers around the world. Some process just a single email account (eg. `postmaster@` of their own company), and some process many thousands of email accounts (eg. CRM services that process their own customer's mailboxes). All have the same subscription. The fixed subscription price covers an unlimited number of EmailEngine instances and an unlimited number of email accounts you can register with EmailEngine. Basically, you are only restricted by your server resources, as more email accounts require more RAM and CPU.
2. The team member count indicates the number of people that can sign up at [postalsys.com](http://postalsys.com/) under the same team account (the first person creates the company account and invites other members of their team with an invite link). Usually, only a few such accounts are needed - for example, one person manages the license keys, and one person provides the billing details.
3. Postal Systems is a tiny company. Doing anything unreasonable would drive away existing and potential customers. There is no way I could afford that.

**Comment by steffenstolze** [comment]
Hey Andris,

thanks a lot for the quick reply!

I understand that while there might be no _guarantee_ that there will be no change in the license model and the commercial use of EmailEngine, it's highly unlikely. It's basically the same with Nodemailer, right? If you were to decide that it's no longer commercially usable under MIT license, the companies using it would also have to deal with it.

Thank you!

**Comment by andris9** [comment]
Yes, I try to run Postal Systems as a company the same I've managed Nodemailer and my other open-source projects. Nodemailer is more than ten years old now, and I don't think any other mainstream Node.js library is as backward compatible as Nodemailer. It supports all Node.js versions back to v6.0.0, which was released already in 2016. I actually tried to change the license for Nodemailer around six years ago (from MIT to EUPL) but had to revert after 2 or 3 months as the license change created too many issues to deal with, so I know it is something I do not want to deal with.
